README.md

```diff
@@ -237,53 +254,29 @@ $ cd modality-probe-cli
 $ cargo install --path .
 $ cd ../
 $ modality-probe manifest-gen \
-    --lang rust \
-    --file-extension rs \
+    --lang c \
+    --file-extension c \
     --component-name example-component \
     --output-path example-component \
-    examples/rust-example
+    examples/c-example
 ```
 
 Next, we'll want to generate the source code that gives those symbols
 we discussed in the code snippet examples in the previous section
 their definitions. To do that, we'll use `header-gen`:
 
 ```shell
+$ mkdir -p /examples/c-example/include
```

Comment:
`mkdir` is no longer required, `modality-probe` CLI will create them as needed.

Reply:
:+1: Also updated in `modality-probe-capi/README.md`